tutorials/ph-ug-temp/ph-ug-temp.js.txt

```diff
@@ -0,0 +1,56 @@
+// Objectives: We want to pull Land Surface Temperature (LST) day data 
+// for 1 year in Uganda.
+//
+// We want to be able to describe the data using a time series map using a chart.
+//
+// We want to export a raster to use in public health research.
+
+
+// Import vector file with country boundaries from Earth Engine
+var dataset = ee.FeatureCollection('USDOS/LSIB_SIMPLE/2017'); // All countries
+var uganda_border = dataset.filter(ee.Filter.eq("country_na","Uganda")); 
+// apply filter where "country names" equals "Uganda"
+print(uganda_border) 
+// print to ensure that new "uganda_border" object just contains polygon for Uganda
+
+  
+//Add region outline to layer - for selected countries

```

Comment:
`//Add` → `// Add` - Be consistent with a single space after `//`

Reply:
Done